dependencies.yaml

```
@@ -70,8 +87,19 @@ files:
       - test_python
 channels:
   - rapidsai
+  - rapidsai-nightly
   - conda-forge
 dependencies:
+  rapids_build_skbuild:
+    common:
+      - output_types: conda
+        packages:
+          - rapids-build-backend >=0.1.1
```

As discussed at https://github.com/rapidsai/cuml/pull/5804#discussion_r1600244946, I think we should have an upper bound here (and all other instances as well):

```suggestion
          - rapids-build-backend >=0.1.1,<0.2.0.dev0
```

---

yep thanks! Pushed this in https://github.com/rapidsai/rmm/pull/1502/commits/a507ae1e4bfb81e596ef77cca2e26aa5c1f6daf9.

I went with `~=0.1.1` for the conda one, just out of minor personal preference. It'll achieve the same effect.

---

blegh forgot the `rmm` conda recipe, added in https://github.com/rapidsai/rmm/pull/1502/commits/dcba4c2d696cf46b8a3d2d073475299df2b3d952